vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java

```diff
@@ -136,7 +135,11 @@ public boolean getIsRunOnGrid() {
      * @return the type
      */
     public String getApplicationType() {
-        return appCbItem._getValue().toString();
+        if (appCbItem._getValue() == null){
+            return "Unknown";
```

Comment:
It is better to return `null` than to create a new constant that will be copy/parted